src/args.c

```diff
@@ -25,7 +25,7 @@ void args_exec_commands(int argc, char **argv)
 	}
 }
 
-void args_parse(int argc, char **argv)
+void args_parse(int argc, char **argv, int override)
```

Comment:
Unless there are reasons for `override` other than calling `config_process_line()` more than once, I suggest we remove it.